pyteal/ir/labelref.py

```diff
@@ -0,0 +1,20 @@
+class LabelReference:
```

Comment:
This class it basically a pass-by-reference string. It was added so that `flattenSubroutines` can easily prefix all the labels in a given subroutine in order to differentiate them.